### ISSUE TITLE
cleanup.sh errors out if ZooKeeper not running

### ISSUE THREAD

**Issue by gerlowskija** [fullstorydev/solr-bench#31]
A line towards the end of cleanup.sh causes the script to error out if no ZooKeeper process is running on the box:

```
$ ./cleanup.sh
kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]
```

...which comes from the script command:

```
kill -9 `jps | grep QuorumPeerMain| cut -f 1 -d " "`
```

Not a major issue, but it'd be a nicer experience if we made this 'kill' command dependent on the jps/grep sub-shell actually producing a pid to terminate.

**Comment by gerlowskija** [comment]
I noticed this issue on branch `ishan/repeatable-jenkins` (as of 1d1a3620548ad6a79fc501c1da249f9c9e372a38).  I went to go open a PR just now against 'master', but noticed that cleanup.sh is drastically different on that branch and doesn't seem to have the same problem.

I guess it's not clear to me what the development workflow is intended to look like on this repo.  I was pointed to the `ishan/repeatable-jenkins` branch by Ishan's comment [here](https://issues.apache.org/jira/browse/SOLR-16531?focusedCommentId=17631216&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17631216) so that's what I've been looking at mostly.  Is `ishan/repeatable-jenkins` a feature branch that will get merged to `master` at some point, or something else?  Is it worth opening PRs against `i/r-j`?

**Comment by gerlowskija** [comment]
I found a few other small bugs in code specific to `i/r-j` (the most signficant of which is that the `startup + recovery` computed for restart-solr-node tasks is incorrect).  Can open PRs for them as well, once I get a little clarification on what the development workflow is here. (i.e. Is `i/r-j` just a personal branch that @chatman has?  Is it a feature branch that's worth fixing bugs on? etc.)

**Comment by chatman** [comment]
Hi Jason, apologies I couldn't respond earlier due to a wrist injury. I'll
get to work soon now. Thanks for your patience.

**Comment by gerlowskija** [comment]
No worries, and hope your wrist heals up.

Don't rush getting back to work for my sake - I've got the fixes locally so I'm fine.  I just wanted to offer them upstream if the intention is that the stuff on `ishan/repeatable-jenkins` is going to be around for some time or get merged into main, or whatever.

**Comment by chatman** [comment]
> Is it a feature branch that's worth fixing bugs on? etc.)

Yes. It will be merged to main/master soon. It is WIP work on the following PR:
https://github.com/fullstorydev/solr-bench/pull/30

Thanks for the startup time calculation bug, fixed it. Feel free to commit directly to that branch, or raise PR against it.
